kotlin-checks-test-sources/src/main/kotlin/checks/TooLongLambdaCheckSample.kt

```diff
@@ -0,0 +1,41 @@
+package checks
+
+abstract class TooLongLambdaCheckSample {
+
+    constructor() {
+        println()
+        println()
+    }
+
+    fun example() {
+        println()
+        println()
+    }
+
+    fun compliant() {
+        println()
+    }
```

Comment:
These are all functions and not lambdas, so we're not testing much lambda-check related here. How about adding `{}` ab below in the Noncompliant example?